gateway-sp-comms/src/single_sp.rs

```diff
@@ -740,10 +750,67 @@ impl SingleSp {
     /// Instruct the SP to reset a component.
     ///
     /// Only valid after a successful call to `reset_component_prepare()`.
+    ///
+    /// If `disable_watchdog` is `true`, then any watchdogs associated with the
+    /// reset are disabled.  Otherwise, watchdogs are enabled opportunistically
+    /// (depending on component and MGS protocol version).
     pub async fn reset_component_trigger(
         &self,
         component: SpComponent,
+        disable_watchdog: bool,
     ) -> Result<()> {
+        // MGS protocol version in which SP watchdog messages were added
+        const MGS_WATCHDOG_VERSION: u32 = 12;
```

Comment:
Tiny nit - I wonder if it would make sense to put this inside the `gateway_messages::version` module, so any future explicit version checks start to collect similar constants in one place?

Reply:
Sure, done in 747d750d97ce3beadd1073d7d62c2f093fe1bc5c